src/Switches.h

```diff
@@ -94,7 +95,35 @@ class SwitchOnCycleDuration {
 		}
 		g_cycleStartTime = nextCycleStartTime;
 	}
+
+	inline void execute(int ifd, int buffer_size) {
+		execute();
+	}
+
+};
+
+//=============================================================================
+class SwitchOnDummySend {
+public:
+	// dummy send between two cycles starting point and take starting point of next cycle
+
+	inline void execute(int ifd, int buffer_size) {
+		uint8_t buffer[buffer_size];
+		TicksTime nextCycleStartTime = g_cycleStartTime + g_pApp->m_const_params.cycleDuration;
+		while (!g_b_exit) {
+			if (TicksTime::now() >= nextCycleStartTime) {
+				break;
+			}
+
+			sendto(ifd, buffer, sizeof(buffer), MSG_SYN, (struct sockaddr*)& (g_fds_array[ifd]->server_addr), sizeof(struct sockaddr));
```

Comment:
Currently this dummy send is happening at the max rate allowed which is not  what we expect customers to do in real life. We don't want to send in such a high rate.

Better to have the '--use-dummy' configuration to get a rate, so we can run it with 1000 mps as warmup.
I would recommend having a default rate of 1000mps as --dummy-send-rate, and optionally have user set a different value, like 10kmps